docs/implementation/fhir-data-model.md

```diff
@@ -14,6 +14,7 @@ Parsing a JSON document like:
 {
  "resourceType": "Patient",
  "id": "0",
+ "name": "John Doe",
```

Comment:
name isn't a string. please check the FHIR spec how a HumanName will look

Reply:
No longer necessary after 0.30.1 release.

Reply:
https://hl7.org/fhir/R4B/datatypes.html#HumanName